apps/web/components/ui/UsernameAvailability/PremiumTextfield.tsx

```diff
@@ -1,3 +1,4 @@
+import { StarIcon as StarIconSolid } from "@heroicons/react/solid";
```

Comment:
Here is room for improvement, though, as we're including from a specific library again beyond the abstraction. NIT as we use so little hero-icons. I think you've looked into this already and this was not possible.

Reply:
Yes, a good follow up would be to get rid of heroicons for good